pkg/kubectl/cmd/get/get.go

```diff
@@ -171,6 +173,7 @@ func NewCmdGet(parent string, f cmdutil.Factory, streams genericclioptions.IOStr
 	cmd.Flags().StringVar(&o.Raw, "raw", o.Raw, "Raw URI to request from the server.  Uses the transport specified by the kubeconfig file.")
 	cmd.Flags().BoolVarP(&o.Watch, "watch", "w", o.Watch, "After listing/getting the requested object, watch for changes. Uninitialized objects are excluded if no object name is provided.")
 	cmd.Flags().BoolVar(&o.WatchOnly, "watch-only", o.WatchOnly, "Watch for changes to the requested object(s), without listing/getting first.")
+	cmd.Flags().BoolVar(&o.OutputWatchEvents, "output-watch-events", o.OutputWatchEvents, "Output watch event objects when --watch or --watch-only is used. Existing objects are output as initial ADDED events.")
```

Comment:
Maybe this should be `--watch-events` or `--include-events` (although I could maybe see that being confused with Events).  We're not super strict about prefixing flags though.

Reply:
I like `--output` as a prefix, and since it's only allowed with `--watch` (or `--watch-only`), I thought qualifying it would be clearer

Reply:
`--include-events` sounds like what describe does, separately fetching/printing related events